jupytercalpoly/Jupyter-Bifrost
# Add onboarding column tag

Fix #7 Fix #9
## Changes
- Added onboarding input tag
- Added column tags to the choosing columns section
- Added search icon next to the search bar
- Changed the text and icon style of onboarding 
- Skipped onboarding sections
    - Without mark, with x and y
    - Without mark, with only x
    - With mark, with x and y
    - With mark, with only x
- Added shift enter keyboard shortcut to go to the next page on `column select` section
- Added key bindings to go though the search results/suggested charts with arrow keys and enter to select.
- Added a few error handlings

## Testing
- Call plot() with different combinations of the key arguments
- Search bar should get autofocused
- Type out the column and hit enter
- See the tag with the first search result getting created
- Hitting backspace without search value would delete the most recently created tag
- Press `shift + enter` to go the next page.
